src/Symfony/Component/DependencyInjection/Tests/ContainerTest.php

```diff
@@ -306,11 +306,39 @@ public function testThatCloningIsNotSupported()
         $this->assertFalse($class->isCloneable());
         $this->assertTrue($clone->isPrivate());
     }
+
+    /** @group legacy */
+    public function testUnsetInternalPrivateServiceIsDeprecated()
+    {
+        $c = new ProjectServiceContainer();
+        $c->set('internal', null);
```

Comment:
A test without any assertion is wrong.

You should assert that the deprecation is triggered instead